### PR TITLE
solution

### PR DESCRIPTION
**Pull request by ivanzh13**

### CODE REVIEW

File: app/players/dwarves/dwarf_blacksmith.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        skill_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> None:
+        return "Dwarf blacksmith {0} with skill of the {1} level".format(
+            self.nickname,
+            self._skill_level
+        )
```
Comment (ArsenPidhoretskyi) [inline app/players/dwarves/dwarf_blacksmith.py:18]: Use f-string

Comment (ArsenPidhoretskyi) [reply app/players/dwarves/dwarf_blacksmith.py:18]: Fix in all places

File: app/players/player.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
```
Comment (ArsenPidhoretskyi) [inline app/players/player.py:9]: It returns int. You must specify type which you expect from all child classes

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/players/dwarves/dwarf_blacksmith.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+        self,
+        nickname: str,
+        favourite_dish: str,
+        skill_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
```
Comment (vasylhnatiuk) [inline app/players/dwarves/dwarf_blacksmith.py:11]:
```suggestion
        super().__init__(nickname=nickname, favourite_dish=favourite_dish)
```

Comment (vasylhnatiuk) [reply app/players/dwarves/dwarf_blacksmith.py:11]: use key arguments

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — APPROVED** [review]
Good job

**Review by vasylhnatiuk — APPROVED** [review]
Good job

**Pull request closed without merging**